.github/workflows/nightly-bnb.yml

```diff
@@ -39,16 +39,19 @@ jobs:
           mkdir transformers-clone && git clone https://github.com/huggingface/transformers.git transformers-clone # rename to transformers clone to avoid modules conflict
 
       - name: Run examples on single GPU
+        if: always()
         run: |
           source activate peft
           make tests_examples_single_gpu_bnb
       
       - name: Run core tests on single GPU
+        if: always()
```

Comment:
This is the quick and simple way. Just want to point out:

- `if: always()` will make the step uncancellable. A better approach would be `if: !cancelled()`

Reply:
But I believe we only  want to run the 3 tests only if the step `Pip install` succeed. The condition gets a bit more complex in this case. OK for me if you are happy to keep the current version.

Reply:
Perfect thanks! I will stick to the current plan then :D